tools/autograph-monitor/monitor.go

```diff
@@ -67,37 +61,29 @@ func main() {
 	// prod or autograph dev code signing PKI roots and CA root
 	// certs defined in constants.go
 	conf.env = os.Getenv("AUTOGRAPH_ENV")
+	conf.truststore = x509.NewCertPool()
 	switch conf.env {
 	case "stage":
-		conf.rootHash = firefoxPkiStageRootHash
```

Comment:
Refactoring to just have `truststore` and `depTruststore`. 
Also using arrays for multiple roots.

Reply:
Added a helper function to populate and no longer need root hashes for stage and prod.